Remotion/Core/Collections.Caching.UnitTests/InvalidationTokenBasedCacheDecoratorTest.cs

```diff
@@ -93,21 +94,20 @@ public void GetOrCreateValue_AfterTokenWasInvalidated_RefreshesRevision_ReturnsV
     [Test]
     public void GetOrCreateValue_RetriesClearUntilInvalidationTokenIsCurrent ()
     {
-      var cacheStub = MockRepository.GenerateStub<ICache<object, string>>();
+      var cacheStub = new Mock<ICache<object, string>>();
       var invalidationToken = InvalidationToken.Create();
-      var decorator = new InvalidationTokenBasedCacheDecorator<object, string> (cacheStub, invalidationToken);
+      var decorator = new InvalidationTokenBasedCacheDecorator<object, string> (cacheStub.Object, invalidationToken);
       var key = new object();
       var count = 0;
-      cacheStub.Stub (_ => _.Clear()).WhenCalled (
-          mi =>
-          {
+      cacheStub.Setup (_ => _.Clear()).Callback (
```

Comment:
Formatting: why did this not get formatted as
```
cacheStub
    .Setup (...)
    .Callback (
    ...);
```

Reply:
because in the original code `Stub` and `WhenCalled` are also on the same line